sys/kern/vm_map.c

```diff
@@ -517,6 +523,69 @@ vm_map_t *vm_map_clone(vm_map_t *map) {
   return new_map;
 }
 
+int insert_anon(vm_map_entry_t *ent, size_t off, vm_anon_t **anon,
```

Comment:
* Please add `static`.
* Usually when a pointer to a value is passed to a function in order to return the value, the argument is suffixed with `p`, for instance `anonp`.